packages/validator/src/.defaultsForValidator.js

```diff
@@ -61,7 +61,6 @@ const defaults = {
     'schemas': {
       'invalid_type_format_pair': 'error',
       'snake_case_only': 'off',
-      'description_mentions_json': 'warning',
```

Comment:
I forgot about this before but we also need to remove the two mentions of each rule in the README. I did this for all of the previously converted rules [in my PR](https://github.com/IBM/openapi-validator/pull/375/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5) so you would just need to do it for this rule